setup.py

```diff
@@ -23,6 +25,21 @@
     inc_dirs.append(numpy.get_include())
 
 
+def read(path, encoding='utf-8'):
+    path = os.path.join(os.path.dirname(__file__), path)
+    with io.open(path, encoding=encoding) as fp:
+        return fp.read()
+
+
+def get_install_requirements(path):
+    content = read(path)
+    return [
+        req
+        for req in content.split("\n")
+        if req != '' and not req.startswith('#')
+    ]
```

Comment:
style quibble: I don't like the closing paren being indented that way.  I would probably put the `read` functionality inside the `get_install_requirements` function, since it isn't used anywhere else, and then format the return as
```python
    return [req for req in content.split("\n")
                if req and not req.startswith('#')]
```